src/components/Editor/components/MySteps/components/MyStepsList/components/Step/index.js

```diff
@@ -103,7 +103,7 @@ export default class Step extends Component {
 
     render() {
         return (
-            <div className={classNames(bootstrap.row, step.step)} data-step={this.props.stepNumber}
+            <div id="mathStep" className={classNames(bootstrap.row, step.step)} data-step={this.props.stepNumber}
```

Comment:
shouldn't id be more unique? There are multiple steps. Additionally, I haven't found it's usage